src/app/common/stores/applications/SsvWeb/Operator.store.ts

```diff
@@ -138,6 +138,7 @@ class OperatorStore extends BaseStore {
       declaredOperatorFeePeriod: observable,
       operatorApprovalBeginTime: observable,
       validatorsPerOperatorLimit: action.bound,
+      getOperatorValidatorsLimit: action.bound,
```

Comment:
What is the difference between these two?

```
validatorsPerOperatorLimit: action.bound,
      getOperatorValidatorsLimit: action.bound,
```

Reply:
done